.travis.yml

```diff
@@ -3,13 +3,14 @@ language: go
 go:
   - 1.8
   - 1.9
-  - tip
+  - 1.10.x
+  - 1.11.x
 before_install:
   - openssl aes-256-cbc -k "$super_secret_password" -in parameters.json.enc -out parameters.json -d
 install:
   - go get github.com/satori/go.uuid
 script:
-  - make fmt lint cov
```

Comment:
why lint was removed?

Reply:
Because the lint tool cannot be install in the old environment with some dependency missing.